app/__init__.py

```diff
@@ -59,6 +59,16 @@
 authenticated_service = LocalProxy(lambda: _request_ctx_stack.top.authenticated_service)
 
 
+# Priority lanes feature (FF_PRIORITY_LANES)
+class RedisQueues(object):
+    SMS_BULK = RedisQueue("sms_bulk", process_type="bulk")
+    SMS_NORMAL = RedisQueue("sms", process_type="normal")
+    SMS_PRIORITY = RedisQueue("sms", process_type="priority")
+    EMAIL_BULK = RedisQueue("email", process_type="bulk")
+    EMAIL_NORMAL = RedisQueue("email", process_type="normal")
+    EMAIL_PRIORITY = RedisQueue("email", process_type="priority")
+
```

Comment:
actually just thought of something. this is a class that hasn't been instantiated. We will need to add an instantiation of this class in the init in order to have the same  the queue each time

```
redis_queues = RedisQueue()

```

and then wherever it is called it will have to be:
```
redis_queues.SMS_BULK
```

else we will be calling a new object each time.

Reply:
I also think if this is required ^
then it makes sense to remove `class RedisQueues`

Reply:
Fixed!